AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/LocalVideoConfiguration.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LocalVideoConfiguration.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Configuration for a local video or content share to be sent
+@objcMembers public class LocalVideoConfiguration: NSObject {
+
+    /// The flag to disable/enable simulcast, default to true
+    public var simulcastEnabled: Bool
+    
+    convenience override public init() {
+        self.init(simulcastEnabled: true)
+    }
+    
+    public init(simulcastEnabled: Bool) {
+        self.simulcastEnabled = simulcastEnabled
```

Comment:
Nit: I feel like these inits are unnecessary, and clunky as we add more optional configurations. Lets just set defaults on the class itself.